ios-search-bar-placement-impact-assessment.toml

```diff
@@ -1,2 +1,17 @@
 [experiment]
-end_date = "2022-08-05"
\ No newline at end of file
+end_date = "2022-08-05"
+
+[metrics]
+overall = ["new_profile_activation"]
+
+[metrics.new_profile_activation]
+data_source = "new_profile_activation"
+select_expression = "COALESCE(SUM(activated))"
```

Comment:
@danielkberry for my own understanding, when using `statistics.binomial` we just need a 0 or 1 on client level, right? If that's the case, you don't need the SUM, my table is already one row per client with the 0 or 1. You also only need the coalesce if there is some left join happening somewhere, because the table shouldn't have NULLs.

Reply:
Thanks for the points about `SUM`, I'll take that out. Under the hood Jetstream does some joins to, for example, the table that records which users were enrolled in this experiment so I think defensively we usually just wrap everything in a `COALESCE`